client/components/Carousel.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Slider from 'react-slick';
+
+class Carousel extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      images: this.props.images
+    }
```

Comment:
I doesn't look like this state actually changes at any point within this Component, is there a reason the props are being cast to state? I think they can just stay as props, if the parent Component requires a re-render and passes down a new set of props then it'll re-render this anyway.

Reply:
I'll probably have to refactor this whole carousel thing